### ISSUE TITLE
ANSI codes were not removed when saving linter report to a file as part of an AzDO pipeline

### ISSUE THREAD

**Issue by marcelschork** [SAP/ui5-linter#175]
## Expected Behavior
When saving the results of the linter to a file the ANSI codes should always be removed.

## Current Behavior

When saving the results of the linter to a file within a CI pipeline on AzDO the ANSI codes were not removed. Running the same command on the terminal (MacOS 13.2.1 + zsh/bash) the ANSI codes are removed and everything is fine. 

We are writing the lint results back to a pull request in AzDO and this is how it is looking like (without the sed workaround being applied)

![image](https://github.com/SAP/ui5-linter/assets/63337737/9cd8395c-02b3-4d4d-a8b4-540c87a685e9)

I did not yet check any other CI/CD platform like GitHub Actions, Jenkins, etc. 

## Steps to Reproduce the Issue

1. Take any UI5 application with some ui5lint errors
2. Create a build pipeline in AzDo
```yaml
pool:
  vmImage: 'ubuntu-latest'

steps:
  - script: |
       corepack enable
       corepack prepare pnpm@latest-9 --activate
    displayName: "🛠 Setup pnpm..."
  - script: >
       pnpm install -D @ui5/linter@0.2.5
    displayName: "🛠 Install dependencies..."
  - script: 
       npx ui5lint --details &> $(Build.ArtifactStagingDirectory)/ui5lint-results.txt || true
       # sed 's/\x1b\[[0-9;]*m//g' $(Build.ArtifactStagingDirectory)/ui5lint-results.txt > $(Build.ArtifactStagingDirectory)/ui5lint-results-no-color.txt # workaround
    displayName: "🔍 Run ui5lint..."
  - task: PublishBuildArtifacts@1
    displayName: "📦 Publish build..."
```

## Context

- UI5 linter version: **0.2.5**
- Node.js Version: **20.14.0**
- npm Version: `pnpm 9.4.0`
- OS/Platform: `Ubuntu 22.04`
- Other information regarding your environment *(optional)*: bash environment running on agent with image ubuntu-22.04

**Comment by RandomByte** [comment]
It looks like that environment does not support the formatting/coloring we are doing using [chalk](https://github.com/chalk/chalk/). Such environments should be detected by chalk itself (through [supports-color](https://github.com/chalk/supports-color)). But I recall similar issues with Azure in the past.

If possible, could you test whether setting the environment variable `CI=true` for the ui5lint command resolves this?

**Comment by marcelschork** [comment]
I just tested it. Setting the env variable CI to true and running the linter does not solve the issue. Then I checked the chalk GitHub repo and FORCE_COLOR=0 is what I was looking for. We may can add it to the documentation.

**Comment by RandomByte** [comment]
Glad to hear that you were able to resolve the issue. I'm still not sure whether this is an issue with chalk or Azure, knowing that Azure _can_ display colors in other places. I'll close this issue for now. If more people run into this, we may add something to the documentation.
